draft-ietf-webpush-protocol.xml

```diff
@@ -556,6 +556,95 @@ TTL = 1*DIGIT
           application server.
         </t>
       </section>
+
+      <section anchor="update" title="Updating Push Messages">
+        <t>
+          A push message that has been stored by the push service can be
+          replaced with new content.  If the user agent is offline during the
+          time that the push messages are sent, updating a push message avoids
+          the situation where obsolete or redundant messages are sent to the
+          user agent.
+        </t>
+        <t>
+          Only push messages that have been assigned a topic can be updated.  A
+          push message with a topic replaces any outstanding push message with
+          an identical topic.
+        </t>
+        <t>
+          A push message topic is a string.  A topic is carried in a Topic
+          header field.  A topic is used to correlate push messages sent to the
+          same subscription, it does not convey any other semantics.
+        </t>
+        <t>
+          The grammar for the Topic header field uses the <spanx
+          style="verb">token</spanx> and <spanx
+          style="verb">quoted-string</spanx> rules defined in <xref
+          target="RFC7230"/>.
+        </t>
+        <figure>
+          <artwork type="abnf">
+            <![CDATA[
+Topic = token / quoted-string
+          ]]>
+          </artwork>
+        </figure>
+        <t>
+          Any double quotes from the <spanx style="verb">quoted-string</spanx>
+          form are removed before comparing topics for equality.
+        </t>
+        <t>
+          A push message request creates a new push message resource, but
+          simultaneously deletes any existing message resource that has a
+          matching topic.  In effect, the information that is stored for the
+          push message is updated, but a new resource is created to avoid
+          problems with in flight acknowledgments for the old message.  The push
+          service MAY suppress acknowledgement receipts for the replaced
+          message.
+        </t>
+        <t>
+          This mechanism doesn't allow for stored push messages to be deleted.
+          An application server can send a DELETE request toward a push message
+          URL to remove (or acknowledge) an undelivered message.  Note that a
+          push message with a topic that has a zero or absent time-to-live
+          causes any stored message with the same topic to be removed, though a
+          push message with zero time-to-live could be forwarded to the user
+          agent.
+        </t>

```

Comment:
s/though a push message/though this push message/
s/could be forwarded/could still be forwarded/
Trying to make the sentence easier to grasp.